components/DeletePopup/index.js

```diff
@@ -0,0 +1,62 @@
+import styled from "styled-components";
+
+export default function DeletePopup({ message, onConfirm, onCancel }) {
+  return (
+    <StyledPopupContainer>
+      <StyledPopupContent>
+        <StyledCallText>Are you sure you want to <StyledSpan>delete</StyledSpan> this Preference?</StyledCallText>
+        <StyledButtonContainer>
+          <StyledButton onClick={onConfirm}>Okay</StyledButton>
+          <StyledButton onClick={onCancel}>Cancel</StyledButton>
```

Comment:
I would change the position of the buttons, since the primary action "Okay" usually stands on the right:
```suggestion
          <StyledButton onClick={onCancel}>Cancel</StyledButton>
          <StyledButton onClick={onConfirm}>Okay</StyledButton>
```